TourMate/TourMate/Frontend/Common/ViewFactory/ViewFactory+PlanLogView.swift

```diff
@@ -13,8 +13,22 @@ extension ViewFactory {
 
     func getPlanVersionView<T: Plan>(planDisplayViewModel: PlanDisplayViewModel<T>, plan: T) -> some View {
         let action = plan.versionNumber == 1 ? "created" : "updated"
+
         let username = planDisplayViewModel.getPlanModifier(version: plan.versionNumber)?.name ?? "someone..."
-        return PlanLogDisplayHeader(header: "Plan version \(plan.versionNumber) \(action) by \(username)")
+        let diff = getDiffFromPreviousPlan(plans: planDisplayViewModel.allVersionedPlansSortedDesc, plan: plan)
+
+        return PlanLogDisplayHeader(header: "Plan version \(plan.versionNumber) \(action) by \(username)",
+                                    subheader: diff)
+    }
+
+    private func getDiffFromPreviousPlan<T: Plan>(plans: [T], plan: T) -> String {
+        guard let previousPlan = plans.first(where: { $0.versionNumber < plan.versionNumber }) else {
+            return ""
+        }
+
+        let planDiffUtil = PlanDiffUtil()
+
+       return planDiffUtil.getDiff(plan1: plan, plan2: previousPlan)
```

Comment:
Should `previousPlan` be plan1 and `plan` be plan2? Seems to be reversed

Reply:
Oh yeah I will change it